docs/gui/screens.md

```diff
@@ -408,11 +408,12 @@ When rendering the label, you do **not** need to specify the `leftPos` and `topP
 
 ## Registering an AbstractContainerScreen
 
-To use an `AbstractContainerScreen` with a menu, it needs to be registered. This can be done by calling `register` within the `RegisterMenuScreensEvent` on the [**mod event bus**][modbus].
+To use an `AbstractContainerScreen` with a menu, it needs to be registered. This can be done by calling `register` within the `RegisterMenuScreensEvent` on the [**mod event bus**][modbus], for example in the `ClientModEvents` class.
```

Comment:
You can't really specify a class here since registration differs between users. That's the reason why we link back to the events doc.